tensorflow/python/autograph/converters/logical_expressions_test.py

```diff
@@ -14,6 +14,8 @@
 # ==============================================================================
 """Tests for logical_expressions module."""
 
+import tensorflow.compat.v1 as tf
```

Comment:
I forgot this was an internal test. Use ops.enable_tensor_equality instead, it's in python/framework.

Reply:
No, we shouldn't need any.